### PR TITLE
New logic for user/admin precedence order

### PR DESCRIPTION
**Pull request by joeyglocke**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-app-setup-policies.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Users can change the order of their pinned apps on Teams desktop and mobile clie
 
 #### Does user pinning take precedence
 
-If the app setup policy assigned to the user is changed to block user app pinning, Teams removes any apps pinned to the app bar. If the policy is then changed to allow user app pinning, users must repin their previously pinned apps.
+Admin pins always take precedence. If the **Allow user pinning** option is turned on, then users will retain their pinned apps below admin pinned apps. If the **Allow user pinning** option is turned off, then users will lose their pre-existing pins and only admin pinned apps will be present in the app bar.
```
Comment (JohanFreelancer9) [inline Teams/teams-app-setup-policies.md:160]:
```suggestion
Admin pins always take precedence. If the **Allow user pinning** option is turned on, then users will retain their pinned apps below admin pinned apps. If the **Allow user pinning** option is turned off, then users will lose their pre-existing pins, and only admin-pinned apps will be present in the app bar.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Comment by scanum** [comment]
@joeyglocke Could you implement the suggestions from @JohanFreelancer9

**Comment by joeyglocke** [comment]
I think I did, by committing his suggestions. Let me know!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by joeyglocke** [comment]
@JohanFreelancer9 @scanum is there an ETA for when this will be merged into the public doc? This feature is live so we need this updated ASAP. Thanks!

**Comment by JohanFreelancer9** [comment]
@scanum ?

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**